src/components/RadioButtonGroup/RadioButton.tsx

```diff
@@ -0,0 +1,65 @@
+import { theme } from '../../theme'
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { RadioButtonGroupContext } from './BaseRadioButtonGroup'
+
+interface RadioButtonProps {
+  disabled: boolean
```

Comment:
`disabled` props are usually optional
```suggestion
  disabled?: boolean
```

Reply:
ok